daras_ai_v2/img_model_settings_widgets.py

```
@@ -186,7 +187,7 @@ def controlnet_weight_setting(
     )
 
 
-def num_outputs_setting(selected_models: str | list[str] = None):
+def num_outputs_setting(selected_models: str | list[str] = None, model_enum=None):
```

Ruff (0.14.5)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)